analysis/src/ProcessAttributes.ml

```diff
@@ -5,7 +5,7 @@ let rec findDocAttribute attributes =
   let open Parsetree in
   match attributes with
   | [] -> None
-  | ( {Asttypes.txt = "ocaml.doc" | "ns.doc" | "res.doc"},
+  | ( {Asttypes.txt = "ocaml.doc" | "ocaml.text" | "ns.doc" | "res.doc"},
```

Comment:
The `Js.ml` module top level generate a `ocaml.text` attr. The docgen not find any docstring, see the docgen output for [Js.json](https://github.com/rescript-association/rescript-lang.org/blob/303fdf895fb2854f5616c389f15886044a50f59b/index_data/api/Js.json#L3)

The bsc typed tree for Js.ml

> Important, see the first attr [`bsconfig.config`](https://github.com/rescript-lang/rescript-compiler/blob/c799ba1fa5a7fda6c78fbd34e902f877fb1a58c0/jscomp/others/js.ml#L25)

```
[
  structure_item 
    Tstr_attribute "bs.config"
    [
      structure_item 
        Pstr_eval
        expression 
          Pexp_record
          [
            "flags" 
              expression 
                Pexp_array
                [
                  expression 
                    Pexp_constant PConst_string("-unboxed-types",None)
                  expression 
                    Pexp_constant PConst_string("-w",None)
                  expression 
                    Pexp_constant PConst_string("-49",None)
                ]
          ]
          None
    ]
  structure_item 
    Tstr_attribute "ocaml.text"
    [
      structure_item 
        Pstr_eval
        expression 
          Pexp_constant PConst_string("\n  The Js module mostly contains ReScript bindings to _standard
 JavaScript APIs_\n  like [console.log](https://developer.mozilla.org/en-US/docs/Web/API/Console/log),\
n  or the JavaScript\n  [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Glob
al_Objects/String),\n  [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_
Objects/Date), and\n  [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Globa
l_Objects/Promise)\n  classes.\n\n  It is meant as a zero-abstraction interop layer and directly expose
s JavaScript functions as they are. If you can find your API in this module, prefer this over an equiva
lent Belt helper. For example, prefer [Js.Array2](js/array-2) over [Belt.Array](belt/array)\n\n  ## Arg
ument Order\n\n  For historical reasons, some APIs in the Js namespace (e.g. [Js.String](js/string)) ar
e\n  using the data-last argument order whereas others (e.g. [Js.Date](js/date)) are using data-first.\
n\n  For more information about these argument orders and the trade-offs between them, see\n  [this blo
g post](https://www.javierchavarri.com/data-first-and-data-last-a-comparison/).\n\n  _Eventually, all m
odules in the Js namespace are going to be migrated to data-first though._\n\n  In the meantime, there 
are several options for dealing with the data-last APIs:\n\n  ```rescript\n  /* Js.String (data-last AP
I used with pipe last operator) */\n  Js.log(\"2019-11-10\" |> Js.String.split(\"-\"))\n  Js.log(\"ReSc
ript\" |> Js.String.startsWith(\"Re\"))\n\n  /* Js.String (data-last API used with pipe first operator)
 */\n  Js.log(\"2019-11-10\"->Js.String.split(\"-\", _))\n  Js.log(\"ReScript\"->Js.String.startsWith(\
"Re\", _))\n\n  /* Js.String (data-last API used without any piping) */\n  Js.log(Js.String.split(\"-\"
, \"2019-11-10\"))\n  Js.log(Js.String.startsWith(\"Re\", \"ReScript\"))\n  ```\n  ## Js.Xxx2 Modules\n
\n  Prefer `Js.Array2` over `Js.Array`, `Js.String2` over `Js.String`, etc. The latters are old modules
.\n",None)
    ]
```

Reply:
Nice catch!